dll/win32/comdlg32/filedlg.c

```diff
@@ -3099,6 +3099,7 @@ BOOL FILEDLG95_OnOpen(HWND hwnd)
         /* Attach the file extension with file name*/
         ext = PathFindExtensionW(lpstrPathAndFile);
 #ifdef __REACTOS__
+        if (*ext == UNICODE_NULL)
```

Comment:
Should this also take into account `fodInfos->defext` like the original code does?

Reply:
Yes it should. Thanks.